gfreshii-webapp/src/main/java/com/edbootcamp/restManagers/RESTIngredientManagerImpl.java

```diff
@@ -0,0 +1,63 @@
+package com.edbootcamp.restManagers;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.client.RestTemplate;
+import com.edbootcamp.api.views.IngredientView;
+import com.edbootcamp.view.IngredientViewImpl;
+import com.edbootcamp.api.manager.IngredientManager;
+
+@RestController
+public class RESTIngredientManagerImpl implements IngredientManager{
+
+	private final String REQUEST_URI = "http://localhost:8080/GFreshiiBackend/recipes";
```

Comment:
Never hardcode full URLs anywhere in your application, this should reference should be provided via a configuration.